### ISSUE TITLE
Bootstrap pagination aria-label "Goto [page blah]" should be "Go to [page blah]"

### ISSUE THREAD

**Issue by johncrossman** [bootstrap-vue/bootstrap-vue#2508]
An example of the markup on a paginated list view:
```
aria-label="Goto page 9"
```
A screen reader pronounces "Goto" like Dorothy's dog "Toto".

Thank you!
